### DISCUSSION TITLE
sqlite GENERATED columns not being recognised

### DISCUSSION THREAD

**Discussion by yuphing-ong** [go-jet/jet#297] [Q&A]
I tried a table like this:
```
CREATE TABLE test (
    jvalues TEXT NOT NULL,
    -- virtual column for json extracted column to be indexed
   column1 STRING GENERATED ALWAYS as (JSON_EXTRACT(jvalues, "$.column1")) STORED,
);
```

but go-jet only generates this model (and associated table):

```
type TEST struct {
        Jvalues       string
 }
```
I tried both virtual and stored, in case go-jet needed it to be STORED  but no difference.

Is this a limitation of the driver, or go-jet?

**Comment by houtn11** [comment]
What's the db? Postgres?

**Reply by yuphing-ong** [reply]
sqlite

**Reply by go-jet** [reply]
Hmm, it seems sqlite added generated columns as well in version 3.26.0. I'll move this discussion into issues.
